src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -156,6 +157,28 @@ public function testGetSetAliases()
         $this->assertEquals(array('name1'), $command->getAliases(), '->setAliases() sets the aliases');
     }
 
+    /**
+     * @expectedException InvalidArgumentException
```

Comment:
since phpunit doesn't handle `use` statements, this should be `@expectedException \Symfony\Component\Console\Exception\InvalidArgumentException`